fcc-test-suite-template-master(1)/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <div id="top-page">
```

Comment:
I think it would be better if you used semantic tag i.e instead of `div id="top-page` you could use the `header` tag